src/main.c

```diff
@@ -355,10 +359,10 @@ main_loop(void)
 			exit(1);
 		}
 	}
-
 	/* TODO: set listening socket */
 	debug(LOG_NOTICE, "Created web server on %s", config->gw_address);
 
+	/* If login script is enabled, check if the script actually exists */
```

Comment:
We should do one line comments in one style. `/**/` or `//`

Reply:
Yes, I agree. As I am in favour of having lots of comments, I think having a recommended style is a good thing. My preferred use is // for single line comments.

Reply:
then let's use `//` from here on :)